content/blog/pulumi-cloud-iam-launch/index.md

```diff
@@ -0,0 +1,115 @@
+    *   Provide mechanisms to manage permissions across hundreds or thousands of Pulumi Entities efficiently.
+
+## A Foundation for Zero Trust & Unified Security
+
+Pulumi IAM isn't just another feature; it's a foundational pillar underpinning security across the entire Pulumi Cloud ecosystem, enabling organizations to implement **Zero Trust** principles for their infrastructure management. Modern security models assume breaches will happen and demand rigorous verification for every access request. Static, organization-wide roles no longer suffice where separation of duties, least privilege, and compliance are paramount.
+
+Pulumi IAM addresses these challenges by providing:
+
+*   **Least Privilege Enforcement:** Define precisely *who* can do *what* on *which* specific resources, minimizing the potential impact if credentials or accounts are compromised. This is core to Zero Trust – grant only the minimum necessary access, verified at the point of action.
+*   **Granular Control Across Pulumi:**
+    *   **Infrastructure as Code (IaC):** Apply fine-grained controls over Pulumi Stacks 
+    *   **Secrets Management:** Define specific access levels for Pulumi ESC Environments.
+    *   **Insights:** Manage permissions for Pulumi Insights account settings.
+*   **Secure Automation:** Provide secure, least-privilege tokens and OIDC integration for CI/CD pipelines and automation, drastically reducing the risk associated with over-privileged service accounts.
+*   **Unified, Scalable Governance:** Establish a consistent authorization model that simplifies administration and scales from small teams to complex enterprise environments, ensuring security doesn't hinder velocity.
+
+## Launching Today: Granular Access Tokens via Custom Roles
+
+This vision begins today with the initial phase of Pulumi IAM, enabling you to define **Custom Roles** built from **fine-grained Permissions** and apply them specifically to **Organization Access Tokens**. This initial step provides immediate, significant security benefits, particularly for automation:
+
+*   **True Least Privilege for CI/CD:** Scope pipeline tokens to *only* the actions (e.g., `pulumi up`) and Entities (e.g., `stack: myapp-prod`) they absolutely need.
+*   **Reduced Blast Radius:** If a scoped token is compromised, the potential damage is limited strictly to the permissions defined in its associated role.
+*   **Enhanced Compliance:** Demonstrate precise control over programmatic access to auditors.
+
+## How to Get Started with Granular Access Tokens
+
+Configuring and using Custom Roles for scoped tokens is done via the Pulumi Cloud console:
+
+#### 1. Define a Custom Permission Set (Optional)
+
+* Create reusable sets of fine-grained permissions. Navigate to Organization Settings -> Roles & Permissions -> Permissions.*
+
+* Click "Create", name it (e.g., "Stack Update Only"), select Resource Type (e.g., Stack), check specific permissions (`stack:read`, `stack:update`), and save.
```

Comment:
super nitpick but the [style guide](https://github.com/pulumi/docs/blob/master/STYLE-GUIDE.md) suggests using `Select` in place of `Click`
```suggestion
* Select "Create", name it (e.g., "Stack Update Only"), select Resource Type (e.g., Stack), check specific permissions (`stack:read`, `stack:update`), and save.
```